html/rendering/non-replaced-elements/form-controls/input-line-height-computed.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>used value and computed value of 'line-height' on input elements as text entry widgets</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/rendering.html#the-input-element-as-a-text-entry-widget">
+<link rel="help" href="https://drafts.csswg.org/cssom/#resolved-values">
+<link rel="help" href="https://drafts.css-houdini.org/css-typed-om/#computed-stylepropertymapreadonly-objects">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+  input { line-height: 1px; }
+</style>
+<p><input type=text value=text>
+<p><input type=tel value=tel>
+<p><input type=search value=search>
+<p><input type=url value=url>
+<p><input type=email value=email>
+<p><input type=password value=password></p>
```

Comment:
It makes sense to me that `password` is included here, but the spec seems to imply that it should be excluded right now.

This special clamping behavior is discussed in the first paragraph of https://html.spec.whatwg.org/multipage/rendering.html#the-input-element-as-a-text-entry-widget seems to be talking about just a specific list of `type` values which *does not* include password.

I see you're editing that prose in https://github.com/whatwg/html/commit/c6440d4aedff2f3d167bfe3127f5c5662be6a0ab - maybe it'd be worth extending that edit such that it includes `password` as well, for the clamping bit?  That would make this part of the test valid, I think.  (E.g. *Additionally: for these types as well as for `type="password", [...stuff about clamping...]*)